kubera/authentication/rest_test.go

```diff
@@ -0,0 +1,28 @@
+package rest
```

Comment:
In `rest.go` and `rest_test.go ` why we are using same `package rest`, any specific reason

Reply:
@rohan2794 pls goto the [Package names](https://blog.golang.org/package-names) Page , u ll get a clear view on Packages

Reply:
In docs i find below statements in package naming convention in Go
`Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64.`

Reply:
`"encoding/base64"`  is a standard golang library package . like this package importing  im also not clear .  
please checkout (about_package_Blog)[https://medium.com/rungo/everything-you-need-to-know-about-packages-in-go-b8bac62b74cc] .